### ISSUE TITLE
fds+openmpi execution randomly stalls

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1202]
```
Application Version: 5.4.0 Parallel
SVN Revision Number:4384
Compile Date:(15 ju 2009) 
Operating System: Linux Ubuntu 64bits (quad-core  proc)

Describe details of the issue below:

Hi,

I have installed intel fortran and openmpi on
ubuntu 64bit system. I successfully compiled
fds sources using mpi_intel_linux_64 makefile.
The computation has been run using the following command:
mpiexec -n 4 fds5_mpi_intel_linux_64 EXEMPLE.PNISI.fds

Problem is that hazardous stalls occur. First time at iteration 8.
I stopped and runned the code using the same command and the 
execution looked normal (I controlled smv output, no problems) 
this time the execution stalled after 3H of CPU time.  Below
are the two last iteration outputs from .out report. 
During stalling there is still a full activity from all 4 
processors. After 16H of stalling there was no new iteration,
even though all of the 4 cores still show full activity.

I have "reniced" processes in order to give them full priority
over system tasks but it didn't change anything.

Thank you very much for your time and attention
Regards

gregoire

    Time Step   12800   October 19, 2010  17:58:07
       ----------------------------------------------
       Mesh    1, Cycle   12800
       CPU/step:     0.859 s, Total CPU:      3.08 hr
       Time step:  0.01880 s, Total time:   267.55 s
       Max CFL number:  0.20E+00 at ( 36, 44, 26)
       Max divergence:  0.22E+00 at (  5, 58, 19)
       Min divergence: -0.23E+00 at (  7, 30, 20)
       Max div. error:  0.76E-02 at ( 36, 48, 26)
       Radiation Loss to Boundaries:         5.244 kW
       Mesh    2, Cycle   12800
       CPU/step:     0.862 s, Total CPU:      3.10 hr
       Time step:  0.01880 s, Total time:   267.55 s
       Max CFL number:  0.24E+00 at ( 17, 32, 26)
       Max divergence:  0.17E+00 at ( 29, 60, 21)
       Min divergence: -0.17E+00 at ( 28, 60, 22)
       Max div. error:  0.23E-01 at ( 37, 39, 26)
       Total Heat Release Rate:              0.056 kW
       Radiation Loss to Boundaries:         8.287 kW
       Mesh    3, Cycle   12800
       CPU/step:     0.873 s, Total CPU:      3.14 hr
       Time step:  0.01880 s, Total time:   267.55 s
       Max CFL number:  0.86E+00 at ( 21, 26, 23)
       Max divergence:  0.44E+01 at ( 20, 27, 25)
       Min divergence: -0.13E+01 at ( 21, 28, 24)
       Max div. error:  0.23E-01 at (  1, 39, 26)
       Total Heat Release Rate:            176.071 kW
       Radiation Loss to Boundaries:        67.224 kW
       Mesh    4, Cycle   12800
       CPU/step:     0.724 s, Total CPU:      2.61 hr
       Time step:  0.01880 s, Total time:   267.55 s
       Max CFL number:  0.17E+00 at ( 51,  3, 28)
       Max divergence:  0.27E-01 at ( 47, 11, 11)
       Min divergence: -0.16E-01 at ( 49, 10, 11)
       Max div. error:  0.11E-02 at ( 47, 11,  1)
       Radiation Loss to Boundaries:         0.021 kW

       Time Step   12900   October 19, 2010  17:59:35
       ----------------------------------------------
       Mesh    1, Cycle   12900
       CPU/step:     0.854 s, Total CPU:      3.11 hr
       Time step:  0.01880 s, Total time:   269.43 s
       Max CFL number:  0.19E+00 at ( 32, 47, 26)
       Max divergence:  0.22E+00 at (  5, 58, 19)
       Min divergence: -0.24E+00 at (  7, 30, 20)
       Max div. error:  0.54E-02 at ( 36, 56, 21)
       Radiation Loss to Boundaries:         5.348 kW
       Mesh    2, Cycle   12900
       CPU/step:     0.858 s, Total CPU:      3.12 hr
       Time step:  0.01880 s, Total time:   269.43 s
       Max CFL number:  0.24E+00 at ( 37, 31, 26)
       Max divergence:  0.19E+00 at ( 30, 59, 21)
       Min divergence: -0.19E+00 at ( 28, 60, 22)
       Max div. error:  0.29E-01 at ( 37, 45, 26)
       Total Heat Release Rate:              0.041 kW
       Radiation Loss to Boundaries:         8.386 kW
       Mesh    3, Cycle   12900
       CPU/step:     0.869 s, Total CPU:      3.16 hr
       Time step:  0.01880 s, Total time:   269.43 s
       Max CFL number:  0.87E+00 at ( 21, 26, 23)
       Max divergence:  0.35E+01 at ( 22, 25, 25)
       Min divergence: -0.11E+01 at ( 22, 28, 24)
       Max div. error:  0.29E-01 at (  1, 45, 26)
       Total Heat Release Rate:            179.743 kW
       Radiation Loss to Boundaries:        68.541 kW
       Mesh    4, Cycle   12900
       CPU/step:     0.720 s, Total CPU:      2.63 hr
       Time step:  0.01880 s, Total time:   269.43 s
       Max CFL number:  0.16E+00 at ( 38, 10, 10)
       Max divergence:  0.18E-01 at ( 37, 11, 11)
       Min divergence: -0.11E-01 at ( 38, 10, 11)
       Max div. error:  0.11E-02 at ( 44, 10,  1)
       Radiation Loss to Boundaries:         0.022 kW

```

Original issue reported on code.google.com by `mcgrok` on 2010-10-20 08:19:23

**Comment by gforney** [comment]
```
sorry I have posted three times due to google errors 
please remove the duplicated issues
```

Original issue reported on code.google.com by `mcgrok` on 2010-10-20 08:29:34

**Comment by gforney** [comment]
```
I don't know what I can do to help. If the problem does not occur at the exact same
point in the calculation, it usually means that the problem is with your system, not
FDS. My only suggestion is to compile the latest version of the code and see if the
problems persist. Let me know what happens.
```

Original issue reported on code.google.com by `mcgratta` on 2010-10-20 13:38:59

**Comment by gforney** [comment]
```
Thank you I will give it a try
Our sys engineer think about a code issue.
Like you I was thinking about some compilation or system
issues, possibly related to openmpi but it seems like
some people succeeded in using open-mpi instead of the
older version lam.

i keep you up about results

```

Original issue reported on code.google.com by `mcgrok` on 2010-10-20 13:58:05

**Comment by gforney** [comment]
```
Issue 1228 has been merged into this issue.
```

Original issue reported on code.google.com by `mcgratta` on 2010-11-01 21:51:34

**Comment by gforney** [comment]
```
Any progress?
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-06 21:02:13

**Comment by gforney** [comment]
```
Sorry, I left this issue aside for a while

I cannot access to svn repository with ubuntu ,
for both command line or kdesvn gui svn client
I tried 
svn co http://fds-smv.googlecode.com/svn/trunk/FDS/trunk FDS-SMV
svn checkout http://fds-smv.googlecode.com/svn/trunk/ fds-smv-read-only
(502 bad gateway)
Thanks for your help


```

Original issue reported on code.google.com by `mcgrok` on 2010-12-07 10:06:40

**Comment by gforney** [comment]
```
Those commands are correct. What if you were to use something like SmartSVN from a Windows
PC?
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-07 13:24:36

**Comment by gforney** [comment]
```
I will try from windows asap,
thank you
```

Original issue reported on code.google.com by `mcgrok` on 2010-12-07 14:09:20

**Comment by gforney** [comment]
```
SmartSVN has failed as well : 

connection refused by the server OPTIONS request failed on
'/svn/trunk/FDS/trunk

I saw a dozen posts about it with no solution, is it because
I'm behind a proxy? (I specified it under smartsvn, without success)

This process is pretty much complicated, I guess some important steps are missing 
in the wiki http://code.google.com/p/fds-smv/wiki/Accessing_Subversion_Repository

```

Original issue reported on code.google.com by `mcgrok` on 2010-12-07 16:09:57

**Comment by gforney** [comment]
```
I don't know enough about SVN repositories. In any case, I suggest you get some help
with these various issues. I cannot debug Ubuntu, GoogleCode, etc. Only FDS. And there
are many people who can compile, run, access the repository. Being behind a proxy might
be a problem. Added security on your system might be another.
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-07 16:35:36

**Comment by gforney** [comment]
```
Hello,

I have solved the SVN issue with tortoise via normal internet connection (without proxy).

Some errors occured while trying to compile version 5.5.3 parallel:

>make mpi_intel_linux_64
mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/dump.f90
../FDS_Source/dump.f90(2295): error #7002: Error in opening the compiled module file.
 Check INCLUDE paths.   [SCARC_SOLVER]    

echo $INCLUDE :/opt/intel/Compiler/11.1/073/ipp/em64t/include:
               /opt/intel/Compiler/11.1/073/mkl/include:

older version 5.4.0 parallel (4384) compile with no error on the same configuration

Thanks for your help   

```

Original issue reported on code.google.com by `mcgrok` on 2010-12-09 16:15:23

**Comment by gforney** [comment]
```
Are you using the Makefile that is in the Repository? If so, delete all the .o and .mod
files and try again. Perhaps making everything from scratch will help. Also, sometimes
if you just type the command again, it works. The reason is that the compiler is working
in parallel, and sometimes routines are compiled out of order.
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-09 19:46:32

**Comment by gforney** [comment]
```
mmmh
I used Makefile from FDS_Compilation folder from svn repository,
I tried to switch positions in compilation order with no success
I see a lot of issues related to solvers, preconditioners so
may be  intel library like MKL is missing or corrupted, 
I will try to search this way

```

Original issue reported on code.google.com by `mcgrok` on 2010-12-10 12:51:11

**Comment by gforney** [comment]
```
What version of Intel Fortran do you have?
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-10 12:55:24

**Comment by gforney** [comment]
```
Intel(R) Fortran Intel(R) 64 Compiler Professional for applications running on Intel(R)
64, Version 11.1    Build 20100806 Package ID: l_cprof_p_11.1.073
Copyright (C) 1985-2010 Intel Corporation.  All rights reserved.


```

Original issue reported on code.google.com by `mcgrok` on 2010-12-10 13:01:05

**Comment by gforney** [comment]
```
This is what my compilation looks like. What do you see when you try to build.

[mcgratta@fire78 mpi_intel_linux_64]$ ./make_fds.csh
Using Fortran located at: /opt/intel/Compiler/11.1/069
Building mpi_intel_linux_64
icc -c -m64 -O -Dpp_noappend -vec_report0 ../../FDS_Source/isob.c
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/prec.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/mpip.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/cons.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/devc.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/type.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/pois.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/mesh.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/smvv.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/func.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/irad.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/ieva.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/part.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/ctrl.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/turb.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/hvac.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/fire.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/radi.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/evac.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/vege.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/wall.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/init.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/scrc.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/mass.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/velo.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/dump.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/read.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/divg.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/pres.f90
mpif77 -c -m64 -O3 -vec_report0 ../../FDS_Source/main.f90
mpif77 -m64 -O3 -vec_report0 -o fds5_mpi_intel_linux_64 isob.o prec.o mpip.o smvv.o
cons.o devc.o type.o mesh.o func.o irad.o ieva.o pois.o scrc.o radi.o evac.o part.o
vege.o ctrl.o turb.o dump.o hvac.o read.o mass.o wall.o fire.o divg.o init.o velo.o
pres.o main.o

```

Original issue reported on code.google.com by `mcgratta` on 2010-12-10 13:13:58

**Comment by gforney** [comment]
```
I use bash and I can't run csh script
The command "make mpi_intel_linux_64" worked well with FDS 5.4.0,
compilation commands are exactly the same as yours before failure 

I sourced both $IFORT_COMPILER11/bin/ifortvars.sh and 
$IFORT_COMPILER11/bin/intel64/ifortvars_intel64.sh unsuccessfully

I tried with command mpif77.openmpi same error. Open mpi
executable are already accessible from /usr/bin ()

an example :
@VM-modelisations-/home/gpianet/BOULOT/FDS/SVN/00.FDS/FDS_Compilation             

> make mpi_intel_linux_64                                                         

icc -c -m64 -O -Dpp_noappend -vec_report0 ../FDS_Source/isob.c                    

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/prec.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/mpip.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/smvv.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/cons.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/devc.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/type.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/mesh.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/func.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/irad.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/ieva.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/pois.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/radi.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/evac.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/part.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/vege.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/ctrl.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/turb.f90                            

mpif77 -c -m64 -O3 -vec_report0 ../FDS_Source/dump.f90                            

../FDS_Source/dump.f90(2295): error #7002: Error in opening the compiled module file.
 Check INCLUDE paths.   [SCARC_SOLVER]                                            

USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&


----^                                                                             

../FDS_Source/dump.f90(3080): error #7002: Error in opening the compiled module file.
 Check INCLUDE paths.   [SCARC_SOLVER]                                            

USE SCARC_SOLVER, ONLY: SCARC_METHOD, SCARC_CAPPA, SCARC_NIT, SCARC_RES           

----^                                                                             

../FDS_Source/dump.f90(2768): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_METHOD]                                                         

   WRITE(LU_OUTPUT,'(A,A10)')   '   global solver:     ', SCARC_METHOD            

----------------------------------------------------------^                       

../FDS_Source/dump.f90(2769): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_METHOD]                                                         

   SELECT CASE(SCARC_METHOD)                                                      

---------------^                                                                  

../FDS_Source/dump.f90(2769): error #6608: In a CASE statement, the case-expr must
be of type INTEGER, CHARACTER, or LOGICAL.   [SCARC_METHOD]                       

   SELECT CASE(SCARC_METHOD)                                                      

---------------^                                                                  

../FDS_Source/dump.f90(2770): error #6611: The case-value must be of the same type
as the case-expr.   ['CG']                                                        

      CASE('CG')                                                                  

-----------^                                                                      

../FDS_Source/dump.f90(2771): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_CG_PRECON]                                                      

         WRITE(LU_OUTPUT,'(A,A10)')  '   preconditioner:    ', SCARC_CG_PRECON    

---------------------------------------------------------------^                  

../FDS_Source/dump.f90(2772): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_CG_PRECON]                                                      

         IF (SCARC_CG_PRECON=='MG') THEN                                          

-------------^                                                                    

../FDS_Source/dump.f90(2773): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_SM_PRECON]                                                      

            WRITE(LU_OUTPUT,'(A,A10)') '   MG-smoother:       ', SCARC_SM_PRECON  

-----------------------------------------------------------------^                

../FDS_Source/dump.f90(2775): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_CG_NIT]                                                         

         WRITE(LU_OUTPUT,'(A,I10)')   '   max iterations:    ', SCARC_CG_NIT      

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2776): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_CG_EPS]                                                         

         WRITE(LU_OUTPUT,'(A,E10.2)') '   stopping accuracy: ', SCARC_CG_EPS      

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2777): error #6611: The case-value must be of the same type
as the case-expr.   ['BICG']                                                      

      CASE('BICG')                                                                

-----------^                                                                      

../FDS_Source/dump.f90(2778): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_BICG_PRECON]                                                    

         WRITE(LU_OUTPUT,'(A,A10)')  '   preconditioner:    ', SCARC_BICG_PRECON  

---------------------------------------------------------------^                  

../FDS_Source/dump.f90(2779): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_BICG_PRECON]                                                    

         IF (SCARC_BICG_PRECON=='MG') THEN                                        

-------------^                                                                    

../FDS_Source/dump.f90(2780): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_SM_PRECON]                                                      

            WRITE(LU_OUTPUT,'(A,A10)') '   MG-smoother:       ', SCARC_SM_PRECON  

-----------------------------------------------------------------^                

../FDS_Source/dump.f90(2782): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_BICG_NIT]                                                       

         WRITE(LU_OUTPUT,'(A,I10)')   '   max iterations:    ', SCARC_BICG_NIT    

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2783): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_BICG_EPS]                                                       

         WRITE(LU_OUTPUT,'(A,E10.2)') '   stopping accuracy: ', SCARC_BICG_EPS    

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2784): error #6611: The case-value must be of the same type
as the case-expr.   ['MG']                                                        

      CASE('MG')                                                                  

-----------^                                                                      

../FDS_Source/dump.f90(2785): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_SM_PRECON]                                                      

         WRITE(LU_OUTPUT,'(A,A10)')  '   preconditioner:    ', SCARC_SM_PRECON    

---------------------------------------------------------------^                  

../FDS_Source/dump.f90(2786): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_MG_NIT]                                                         

         WRITE(LU_OUTPUT,'(A,I10)')   '   max iterations:    ', SCARC_MG_NIT      

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2787): error #6406: Conflicting attributes or multiple declaration
of name.   [SCARC_MG_EPS]                                                         

         WRITE(LU_OUTPUT,'(A,E10.2)') '   stopping accuracy: ', SCARC_MG_EPS      

----------------------------------------------------------------^                 

../FDS_Source/dump.f90(2295): error #6580: Name in only-list does not exist.   [SCARC_METHOD]

USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&


------------------------^
../FDS_Source/dump.f90(2295): error #6580: Name in only-list does not exist.   [SCARC_CG_PRECON]
USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&
-------------------------------------^
../FDS_Source/dump.f90(2295): error #6580: Name in only-list does not exist.   [SCARC_BICG_PRECON]
USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&
-----------------------------------------------------^
../FDS_Source/dump.f90(2295): error #6580: Name in only-list does not exist.   [SCARC_SM_PRECON]
USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&
------------------------------------------------------------------------^
../FDS_Source/dump.f90(2295): error #6580: Name in only-list does not exist.   [SCARC_CG_EPS]
USE SCARC_SOLVER, ONLY: SCARC_METHOD,SCARC_CG_PRECON,SCARC_BICG_PRECON, SCARC_SM_PRECON,SCARC_CG_EPS,&
----------------------------------------------------------------------------------------^
../FDS_Source/dump.f90(2296): error #6580: Name in only-list does not exist.   [SCARC_BICG_EPS]
                        SCARC_BICG_EPS,SCARC_MG_EPS,SCARC_CG_NIT,SCARC_BICG_NIT,SCARC_MG_NIT
------------------------^
../FDS_Source/dump.f90(2296): error #6580: Name in only-list does not exist.   [SCARC_MG_EPS]
                        SCARC_BICG_EPS,SCARC_MG_EPS,SCARC_CG_NIT,SCARC_BICG_NIT,SCARC_MG_NIT
---------------------------------------^
../FDS_Source/dump.f90(2296): error #6580: Name in only-list does not exist.   [SCARC_CG_NIT]
                        SCARC_BICG_EPS,SCARC_MG_EPS,SCARC_CG_NIT,SCARC_BICG_NIT,SCARC_MG_NIT
----------------------------------------------------^
../FDS_Source/dump.f90(2296): error #6580: Name in only-list does not exist.   [SCARC_BICG_NIT]
                        SCARC_BICG_EPS,SCARC_MG_EPS,SCARC_CG_NIT,SCARC_BICG_NIT,SCARC_MG_NIT
-----------------------------------------------------------------^
(6488): catastrophic error: Too many errors, exiting
compilation aborted for ../FDS_Source/dump.f90 (code 1)
make: *** [dump.o] Erreur 1


```

Original issue reported on code.google.com by `mcgrok` on 2010-12-10 14:14:58

**Comment by gforney** [comment]
```
Are you sure you have an updated repository? I do not see the compilation of scrc.f90,
which is a new routine in FDS 5.5.x
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-10 14:42:21

**Comment by gforney** [comment]
```
Sorry I unintentionally started from an older svn directory...

You were right, I though it was the last version but the scrc.f90 is missing

My mistake, Thanks for your time I will compile 
the latest sources and check again the current issue.

Have a nice week end , regards

Greg
```

Original issue reported on code.google.com by `mcgrok` on 2010-12-10 14:57:16

**Comment by gforney** [comment]
```
Keep in mind that the current source code is FDS 6. If you just want to have a version
of 5.5.3, dial the repository back to 7031.
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-10 15:02:26

**Comment by gforney** [comment]
```
Thank you very much,

version 7313 compiled with success
If any, I will report execution issue asap

Greetings
```

Original issue reported on code.google.com by `mcgrok` on 2010-12-10 15:30:23

**Comment by gforney** [comment]
```
Hello,
Same issue occurred during the weekend
Calculation stalled at Time Step    3600   December 11, 2010  01:55:50
I stopped calculation on December 13.
Until I stopped it, CPU time was significantly used, as if the code 
was still running in some endless loop.
It is likely some system issue, but I just can't figure how to solve it
(I use Linux 64 installed on a multicore system through a vmware machine)
Thank you very much for your suggestions
Greetings,


```

Original issue reported on code.google.com by `mcgrok` on 2010-12-13 15:11:56

<hr>
- _Attachment: [EXEMPLE.out](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1230/comment-22/EXEMPLE.out)_

**Comment by gforney** [comment]
```
If you run this exact same executable again, does it stall at the same point in the
calculation?
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-13 15:26:36

**Comment by gforney** [comment]
```
Well I'll check this point
I keep you up about it
```

Original issue reported on code.google.com by `mcgrok` on 2010-12-13 16:06:13

**Comment by gforney** [comment]
```
Hello,
Using the same executable, same issue but different stall time step
(time step 2000)
Sorry for bothering you, it is certainly due to a system issue
as you mentioned it earlier...
Thanks for your time and attention,
Regards
```

Original issue reported on code.google.com by `mcgrok` on 2010-12-14 16:56:02

<hr>
- _Attachment: [EXEMPLE_PNISI.out](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1230/comment-25/EXEMPLE_PNISI.out)_

**Comment by gforney** [comment]
```
This happens to us from time to time. All it takes is one bad node, one bad switch,
and you get random error. 
```

Original issue reported on code.google.com by `mcgratta` on 2010-12-15 13:10:31
